internals/core_models.py

```diff
@@ -973,7 +974,7 @@ def put(self, notify=True, **kwargs):
   experiment_extension_reason = ndb.StringProperty()
   ongoing_constraints = ndb.StringProperty()
   origin_trial_feedback_url = ndb.StringProperty()
-  anticipated_spec_changes = ndb.StringProperty()
+  anticipated_spec_changes = ndb.StringProperty(repeated=True)
```

Comment:
Let's keep this as a single string value rather than a list of URLs.   The help text asks the user to enter a list of issues, which could be URLs or it could just be an English description.  In other words, I think the existing defect was with the widget type rather than with the DB or POST processing.

Reply:
I have changed it back to a string value so the core_model.py basically remains unchanged in this PR. I also removed the MULTI_URL_FIELD_ATTRS for the `anticipated_spec_changes` field in form-field-spec.js so the field gets rendered as a regular textarea. Please review the changes again, thank you!